pkg/event/kind/webhook/listener.go

```diff
@@ -83,7 +86,16 @@ func (l *WebhookListener) Headers() map[string]string {
 	return l.headers
 }
 
+func (l *WebhookListener) Disabled() bool {
+	return l.disabled
+}
+
 func (l *WebhookListener) Notify(event testkube.Event) (result testkube.EventResult) {
+	if l.disabled {
+		l.Log.With(event.Log()...).Debug("webhook listener is disabled")
+		return testkube.NewFailedEventResult(event.Id, errors.New("webhook listener is disabled"))
```

Comment:
do we need to return an error?

Reply:
The notification failed, even if it was expected to fail, that is why I chose to return a failed event result

Reply:
I would not report an error, because we expect, it would not work

Reply:
Makes sense. I updated it.